pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "crdch-model"
+version = "1.1.5"
+description = "CRDC-H model in LinkML, developed by the Center for Cancer Data Harmonization (CCDH). This Python package contains the dataclasses necessary to build and operate over data objects according to the CRDC-H model."
+readme = "CRDCH_MODEL.md"
+authors = [
+  "Mark A. Miller <MAM@lbl.gov>",
+  "Gaurav Vaidya <gaurav@ggvaidya.com>"
+]
+license = "MIT"
+packages = [
+  { include = "crdch_model.py", from = "crdch_model" }
+]
+include = [
+  "README.md",
+  "model/*",
+  "crdch_model/*"
+]
+
+[tool.poetry.urls]
+"CCDH Website" = "https://ccdh.cancer.gov"
+"CCDH Model" = "https://cancerdhc.github.io/ccdhmodel/"
+"GitHub Repository" = "https://github.com/cancerDHC/ccdhmodel/"
+"Bug Tracker" = "https://github.com/cancerDHC/ccdhmodel/issues"
+
+[tool.poetry.dependencies]
+python = ">=3.7"
+linkml-runtime = "^1.1.2"
```

Comment:
Add `sheet2linkml = "^1.1.0"` as a core dependency as well? Add it via the `poetry add` command.

Reply:
I can't do this yet, since sheet2linkml 1.1.0 requires Python 3.9. Poetry won't let me install it as a dependency on a project that requires Python 3.7 :). Weren't you modifying sheet2linkml so that it could support Python 3.7? You might just need to change the line at https://github.com/cancerDHC/sheet2linkml/blob/ff3f001192210e7643b4fb78b1525ecdac3571a2/pyproject.toml#L33

Reply:
This is the error message I got:
```
  SolverProblemError

  The current project's Python requirement (>=3.7) is not compatible with some of the required packages Python requirement:
    - sheet2linkml requires Python >=3.9,<4.0, so it will not be satisfied for Python >=3.7,<3.9 || >=4.0
  
  Because sheet2linkml (1.1.0) requires Python >=3.9,<4.0
   and no versions of sheet2linkml match >1.1.0,<2.0.0, sheet2linkml is forbidden.
  So, because crdch-model depends on sheet2linkml (^1.1.0), version solving failed.
```

Reply:
This [PR](https://github.com/cancerDHC/sheet2linkml/pull/17) on the [sheet2linkml](https://github.com/cancerDHC/sheet2linkml) repo fixed the codebase so it's compatible with Python versions 3.7 and 3.8. But I guess I forgot to update the Python dependency version in the `pyproject.toml` file. So I created a new [release](https://github.com/cancerDHC/sheet2linkml/releases/tag/v1.1.1) and also published a new version of the [package](https://pypi.org/project/sheet2linkml/) to PyPI.

Reply:
This [PR](https://github.com/cancerDHC/sheet2linkml/pull/17) on the [sheet2linkml](https://github.com/cancerDHC/sheet2linkml) repo fixed the codebase so it's compatible with Python versions 3.7 and 3.8. But I guess I forgot to update the Python dependency version in the `pyproject.toml` file. So I created a new [release](https://github.com/cancerDHC/sheet2linkml/releases/tag/v1.1.1) and also published a new version of the [package](https://pypi.org/project/sheet2linkml/) to PyPI.

Reply:
You should use `sheet2linkml = ^1.1.1` now.

Reply:
Yup, that worked perfectly! Thanks so much, Sujay!